### DISCUSSION TITLE
formatting datetime fields

### DISCUSSION THREAD

**Discussion by Loris-F** [Budibase/budibase#5656] [Help]
Hello I am building some firsts budibase app for showing some data.
Using Budibase on Debina 11 with Docker, connection to database Postgres v13.
I have some field TIMESTAMP that wil be show in form of ISO format:
"data": "2022-03-21T00:00:00.000Z"
How can I format to different dysplay, i.e "DD/MM/YYY" without time?
I have tried to change the type of field but always same result with TIME DATE DATETIME.
The field is show in Table component.
Thanks

**Comment by melohagan** [comment]
Hi @Loris-F 

You can create a new formatted display column with formulas: https://docs.budibase.com/docs/formula

Where the handle bar would be: `{{ date TIMESTAMP "DD/MM/YYYY" }}`
